### DISCUSSION TITLE
Certificate and Signature issue when trying to install regolith

### DISCUSSION THREAD

**Discussion by HugoLuskas** [regolith-linux/regolith-desktop#816] [Q&A]
Hey,

I'm trying to install regolith desktop fresh on a Ubuntu 20.04 and I'm getting this when I go through the "sudo apt update" step. Would appreciate any help!

Err:33 https://regolith-desktop.org/release-ubuntu-focal-amd64 focal Release                                                                                    
  Certificate verification failed: The certificate is NOT trusted. The certificate chain uses expired certificate.  Could not handshake: Error in the certificate verification. [IP: 104.238.213.207 443]
Err:12 http://packages.cloud.google.com/apt cloud-sdk InRelease                   
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B53DC80D13EDEF05
Reading package lists... Done     
E: The repository 'https://regolith-desktop.org/release-ubuntu-focal-amd64 focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

**Comment by comkieffer** [comment]
Are you sure your system clock was correct? 

Certificate verification and key verification failing at the same time for repos managed by different organizations suggests an issue on your machine.
